Source/Dafny/Compilers/Compiler-go.cs

```diff
@@ -3603,7 +3613,7 @@ private class SpecialNativeType : UserDefinedType {
         verb = string.Format("build -o \"{0}\"", output);
       }
 
-      var args = string.Format("{0} \"{1}\"", verb, targetFilename);
+      var args = string.Format("{0} \"{1}\"", verb, targetFilename) + DafnyOptions.O.ArgsStringExtra;
```

Comment:
This isn't safe; arguments shouldn't be concatenated as strings

Reply:
That's unfortunately the only way to launch a `ProcessInfo`:

https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.processstartinfo.-ctor?view=net-6.0#system-diagnostics-processstartinfo-ctor(system-string-system-string)

Reply:
Thankfully that's not right :) Here's an example of how to do it: https://github.com/dafny-lang/dafny/blob/5058d5a7dab45c1743ce8a254619ad030b26b0d5/Source/AutoExtern.Test/Tests.cs#L16-L31

Reply:
Oh that's very interesting. I changed the existing behavior to also adapt to this.